devstack/README-NFP-Base-Mode

```diff
@@ -0,0 +1,78 @@
+
+Fresh Installation Steps:
+=========================
+
+(1) Clone stable mitaka devstack.
+    # git clone https://git.openstack.org/openstack-dev/devstack -b stable/mitaka
+
+(2) Get local.conf.nfp from NFP devstack patch and copy to devstack directory
+    as local.conf
+    # cd devstack/
+    # wget -O local.conf.zip https://review.openstack.org/cat/334998,4,devstack/local.conf.nfp
+    # unzip local.conf.zip
+    # mv local.conf*.nfp local.conf
+    # rm local.conf.zip
+
+(3) Configure local.conf
+    # Modify the GBPSERVICE_BRANCH to point to the top patch in devstack changeset refs/changes/45/309145
```

Comment:
Didn't look into the README(s). Need to review them according to the latest changes.